src/plugins/workspace/public/components/global_search/page_item.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiHighlight, EuiSimplifiedBreadcrumbs } from '@elastic/eui';
+import {
+  ApplicationStart,
+  ChromeNavLink,
+  ChromeRegistrationNavLink,
+  NavGroupItemInMap,
+  WorkspaceObject,
+} from 'opensearch-dashboards/public';
+import React from 'react';
+import { BehaviorSubject } from 'rxjs';
+import { useObservable } from 'react-use';
+import { WorkspaceTitleDisplay } from '../workspace_name/workspace_name';
+import { WorkspaceUseCase } from '../../types';
+import { NavGroupType } from '../../../../../core/public';
+
+interface Props {
+  currentWorkspace: WorkspaceObject | null;
+  link: ChromeRegistrationNavLink & ChromeNavLink & { navGroup: NavGroupItemInMap };
+  application: ApplicationStart;
+  registeredUseCases$: BehaviorSubject<WorkspaceUseCase[]>;
+  search: string;
+  callback?: () => void;
+}
+
+export const GlobalSearchPageItem = ({
+  link,
+  currentWorkspace,
+  application,
+  registeredUseCases$,
+  search,
+  callback,
+}: Props) => {
+  const availableUseCases = useObservable(registeredUseCases$);
+  const breadcrumbs = [];
+  if (currentWorkspace && link.navGroup.type !== NavGroupType.SYSTEM) {
+    breadcrumbs.push({
+      text: (
+        <WorkspaceTitleDisplay
+          workspace={currentWorkspace}
+          availableUseCases={availableUseCases || []}
+        />
+      ),
+    });
+  } else {
+    breadcrumbs.push({ text: link.navGroup.title });
+  }
+  const appId = link.id.toLowerCase();
+  const isLanding = appId.endsWith('landing');
```

Comment:
Q: Which page ids are ended with `landing`?

Reply:
settings and data administration landing page, those two.